recipes/libmatio/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set version = "1.5.6" %}
+package:
+  name: libmatio
+  version: {{ version }}
+
+source:
+  fn: matio-{{ version }}.tar.gz
+  url: https://github.com/tbeu/matio/releases/download/v{{ version }}/matio-{{ version }}.tar.gz
+  md5: bb53fd06f7ebb74589e9dd684b1b66a6
+
+build:
+  number: 0
+  skip: true     # [win]
+
+requirements:
+  build:
+    - autoconf
+    - automake
+    - libtool
+    - gcc
+    - hdf5
+    - zlib
+
+  run:
+    - hdf5
+    - zlib
+
+test:
+  commands:
+    - test -e $PREFIX/include/matio.h  # [unix]
```

Comment:
> wouldn't their name be different between OSX and Linux? Do I check for static or shared ones?

For shared libraries, OSX uses `.dylib` as the extension and Linux uses `.so`. This is easily resolved by using the `osx` and `linux` selectors.

Static libraries will the the same (both `.a`).

If Windows ever gets added here, we will need to add a couple more, but they have a pattern too.

> both are enabled by the way so `--enable-shared` is useless.

Ah, ok, I saw the flag and not everyone makes the same decision in these cases.